### DISCUSSION TITLE
CKAN pre selecting organization

### DISCUSSION THREAD

**Discussion by Patrick1Rhode** [ckan/ckan#6167] [Ideas]
CKAN allows users to add a dataset straight from an organization by clicking the add dataset button.  The issue is when, e.g., when you are coming from an organization called DCAT, and you click the add dataset button, that organization(DCAT) won't be a pre-selected organization on the metadata adding page.

<img width="1194" alt="organization" src="https://user-images.githubusercontent.com/8822597/122379311-09ec6300-cf67-11eb-864e-fbcb76b6d4b8.png">
<img width="1341" alt="add_dataset" src="https://user-images.githubusercontent.com/8822597/122379454-2a1c2200-cf67-11eb-8d84-67793c1b57a0.png">

**Comment by anuveyatsu** [comment]
@Patrick1Rhode you can implement it with JS code, e.g., pass organization name as a query parameter and then parse it on package create page to preselect the org from the select options.
